junit-jupiter-api/src/main/java/org/junit/jupiter/api/MethodOrderer.java

```diff
@@ -235,6 +238,22 @@ public OrderAnnotation() {
 		 */
 		@Override
 		public void orderMethods(MethodOrdererContext context) {
+			List<? extends MethodDescriptor> descriptors = context.getMethodDescriptors();
+			Map<Integer, String> removeDuplicate = new LinkedHashMap<>();
```

Comment:
I believe the hash set/map is not needed. JUnit would sort the method descriptions anyway, so it would be better to scan over the sorted list and see if there's a set of consequent elements sharing the same `order`.

It would be nice to print all the violations rather than just the first for faster turnaround.